hack/golangci-lint.sh

```diff
@@ -9,16 +9,18 @@ set -e
 # absent.
 
 echo "Linting for GOOS=$GOOS"
-# Special case: for Darwin and Windows only "remote" linting is possible and required.
-if [[ "$GOOS" == "windows" || "$GOOS" == "darwin" ]]; then
+# For Darwin, Windows, and Freebsd only "remote" linting
+# is possible and required.
+case "$GOOS" in
+  windows|darwin|freebsd)
```

Comment:
freebsd has "full" podman support. The libpod code etc.., needs to run/compile on freebsd.

As such I would assume it should lint the same way as podman on linux (minus some linux build tags, apparmor,selinux,seccomp,systemd...)

Reply:
Getting there...